### ISSUE TITLE
Dataframe for non-matrix classification metrics

### ISSUE THREAD

**Issue by fkromer** [scikit-learn-contrib/sklearn-pandas#206]
It would be valuable to have some mapping functionality which takes `y_true`, `y_pred`, `y_score`, `pred_decision` (output of `estimator.decision_function()`), `x` (`AUC` only), `y` (`AUC` only) and converts it into a standardized pandas DataFrame ("non-matrix classification metric dataframe") which could look like follows:

| metric                       | accuracy_score | auc   | ... | zero_one_loss |
|----------------------------|-----------------------|--------|-----|---------------------|
| **estimator**            |                            |         |     |                          |
| LogisticRegression   | float or int           | float |     | float or int         |

`...` could include as well:
- cohen_kappa_score (float)
- dcg_score (float)

**Comment by QuadV** [comment]
This sounds interesting. Could be a good addition to the repo. Can I take it up?
